### ISSUE TITLE
Migrating: Peloton

### ISSUE THREAD

**Issue by valencik** [typelevel/governance#141]
## Migration Checklist

Project Submission Issue: https://github.com/typelevel/governance/issues/136

### For All Projects:

- [ ] Add project to the site by [adding an entry to the projects.yml file](https://github.com/typelevel/typelevel.github.com/blob/development/_data/projects.yml)

### Announcements

These steps are optional but encouraged.

- [ ] Write a blog post announcing the change
- [ ] Post to our social channels (Mastodon)

**Comment by killaitis** [comment]
I added the project to the projects.yml.

https://github.com/typelevel/typelevel.github.com/pull/508

Would appreciate it if someone could write a nice blog post. I'm not good at this at all. :)

**Comment by valencik** [comment]
> projects.yml has been merged now. Is anything else necessary from my side to proceed?

Nope, you're all good. Peloton is now listed on the site: https://typelevel.org/projects/affiliate/

We've been leaving these issues open even if all that remains is the social announcements.
But there's nothing remaining that you need to do. :)

**Comment by reardonj** [comment]
This is still on the site and well past time to announce.
